Mail/Views/New Message/Select Mailbox/SelectComposeMailboxViewModel.swift

```diff
@@ -48,12 +48,17 @@ final class SelectComposeMailboxViewModel: ObservableObject {
     }
 
     func listProfiles() {
```

Comment:
It looks like the previous version of this method was syncronous. It now hides an async task. Can this be an issue if called from SwUI (glitches …)?

Reply:
Good catch. I ensured we call back on main thread